UIKit_HM/UIKit_HM/PizzaOptionsViewController.swift

```diff
@@ -0,0 +1,127 @@
+	private let cheeseLabel = UILabel()
+	private let cheeseSwitcher = UISwitch()
+	private let hamLabel = UILabel()
+	private let hamSwitcher = UISwitch()
+	private let mushroomsLabel = UILabel()
+	private let mushroomsSwitcher = UISwitch()
+	private let olivesLabel = UILabel()
+	private let olivesSwitcher = UISwitch()
+	private let checkButton = UIButton()
+	private let showIngredientsButton = UIButton()
+	
+	// MARK: - Public Properties
+	
+	var pizzaName: String?
+	var currentPizza: String?
+	
+	// MARK: - Private Properties
+	
+	private var orderArray: [String] = []
+	
+	// MARK: - Life cycle
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setUpView()
+	}
+	
+	// MARK: - Private methods
+	private func setUpView() {
+		view.backgroundColor = .white
+		
+		titleLabel.frame = CGRect(x: 60, y: 13, width: 300, height: 45)
+		titleLabel.text = pizzaName
+		titleLabel.textAlignment = .center
+		titleLabel.textColor = .lightGray
+		titleLabel.font = .boldSystemFont(ofSize: 32)
+		view.addSubview(titleLabel)
+		
+		let image = UIImage(named: currentPizza!)
+		pizzaImage = UIImageView(image: image)
+		pizzaImage.frame = CGRect(x: 57, y: 115, width: 300, height: 300)
+		view.addSubview(pizzaImage)
+		
+		cheeseLabel.frame = CGRect(x: 57, y: 496, width: 120, height: 25)
+		cheeseLabel.text =  "Сыр мацарела"
+		view.addSubview(cheeseLabel)
+		
+		cheeseSwitcher.frame = CGRect(x: 308, y: 496, width: 49, height: 31)
+		cheeseSwitcher.isOn = true
+		view.addSubview(cheeseSwitcher)
+		
+		hamLabel.frame = CGRect(x: 57, y: 566, width: 120, height: 25)
+		hamLabel.text = "Ветчина"
+		view.addSubview(hamLabel)
+		
+		hamSwitcher.frame = CGRect(x: 308, y: 566, width: 49, height: 31)
+		hamSwitcher.isOn = false
+		view.addSubview(hamSwitcher)
+		
+		mushroomsLabel.frame = CGRect(x: 57, y: 629, width: 120, height: 25)
+		mushroomsLabel.text = "Грибы"
+		view.addSubview(mushroomsLabel)
+		
+		mushroomsSwitcher.frame = CGRect(x: 308, y: 629, width: 49, height: 31)
+		mushroomsSwitcher.isOn = true
+		view.addSubview(mushroomsSwitcher)
+		
+		olivesLabel.frame = CGRect(x: 57, y: 686, width: 120, height: 25)
+		olivesLabel.text = "Маслины"
+		view.addSubview(olivesLabel)
+		
+		olivesSwitcher.frame = CGRect(x: 308, y: 686, width: 49, height: 31)
+		olivesSwitcher.isOn = false
+		view.addSubview(olivesSwitcher)
+		
+		checkButton.frame = CGRect(x: 57, y: 771, width: 300, height: 45)
+		checkButton.setTitle("Выбрать", for: .normal)
+		checkButton.backgroundColor = .orange
+		checkButton.layer.cornerRadius = 5
+		checkButton.addTarget(self, action: #selector(navigateToCheckout), for: .touchUpInside)
+		view.addSubview(checkButton)
+	}
+	
+	@objc private func navigateToCheckout() {
+		checkExtraFood()
+		
+		let vc = CheckOutViewController()
+		vc.delegate = self
+		vc.orderArray = orderArray
+		
+		let navVC = UINavigationController(rootViewController: vc)
+		navVC.modalPresentationStyle = .fullScreen
+		present(navVC, animated: true, completion: nil)
```

Comment:
```suggestion
		present(navVC, animated: true)
```

везде убери коплишен